src/backend/database_migrations/versions/20221130_222651_mpx_hepc_support.py

```diff
@@ -0,0 +1,37 @@
+"""add MPX and HCV DB support for gisaid/genbank template
+
+
+Create Date: 2022-11-30 22:26:58.178142
+
+"""
+import enumtables  # noqa: F401
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "20221122_222651"
+down_revision = "20221129_012949"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute(
+        """
+        INSERT INTO aspen.pathogens (slug, name) VALUES ('HCV', 'Hepatitis C')
+        """
+    )
+
+    op.execute("select nextval('aspen.pathogen_repo_configs_id_seq')")
+    op.execute("select nextval('aspen.pathogen_repo_configs_id_seq')")
+
+    op.execute(
+        """
+        INSERT INTO aspen.pathogen_repo_configs (prefix, public_repository_id, pathogen_id) VALUES
+             ( 'hcv', (SELECT id from aspen.public_repositories WHERE name='GenBank'), (SELECT id from aspen.pathogens WHERE slug='HCV') ),
+             ( 'hMpxV', (SELECT id from aspen.public_repositories WHERE name='GenBank'), (SELECT id from aspen.pathogens WHERE slug='MPX') )
```

Comment:
I think the genbank prefix is 'MPXV' or 'MpxV' [based on these search results](https://www.ncbi.nlm.nih.gov/labs/virus/vssi/#/virus?SeqType_s=Nucleotide&VirusLineage_ss=Monkeypox%20virus%20(monkeypox),%20taxid:10244&Completeness_s=complete&HostLineage_ss=Homo%20sapiens%20(human),%20taxid:9606) maybe @sidneymbell can clarify?

Reply:
Ooh good catch. You are correct, @jgadling . Weird / annoying that genbank and gisaid differ on this but /shrug

Reply:
do we prefer MPXV or MpxV?